stack/stack.go

```diff
@@ -0,0 +1,189 @@
+package stack
+
+import (
+	"errors"
+	"github.com/Appliscale/perun/context"
+	"github.com/Appliscale/perun/utilities"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/go-ini/ini"
+	"io/ioutil"
+	"os/user"
+	"time"
+)
+
+const dateFormat = "2006-01-02 15:04:05 MST"
+
+//prepare stackinput from template
+func createStackInput(context *context.Context, template *string, stackName *string) cloudformation.CreateStackInput {
+	templateStruct := cloudformation.CreateStackInput{
+		TemplateBody: template,
+		StackName:    stackName,
+	}
+	return templateStruct
+}
+
+//main function to delete stack
+func DestroyStack(context *context.Context) {
+	delStackInput := deleteStackInput(context)
+	session, err := createSession(context, context.Config.DefaultProfile, &context.Config.DefaultRegion)
+	if err != nil {
+		context.Logger.Error(err.Error())
+	}
+	api := cloudformation.New(session)
+	api.DeleteStack(&delStackInput)
+}
+
+// prepare stackinput from --stack
+func deleteStackInput(context *context.Context) cloudformation.DeleteStackInput {
+	name := *context.CliArguments.Stack
+	templateStruct := cloudformation.DeleteStackInput{
+		StackName: &name,
```

Comment:
Let's remove trailing commas from the module (if those are not necessary anyway).

Reply:
I'm not sure how i can fix it. Do you want new variable with this? And use only it?